### ISSUE TITLE
Add title to headerbar

### ISSUE THREAD

**Issue by NathanBnm** [aggalex/Wineglass#37]
I think you have to put the name of the app centred into the header-bar. Maybe you can pull the '+' button to the right next to the full-screen button ?

**Comment by aggalex** [comment]
I always opt for minimalism, and simplicity. The app's name on the headerbar is quite useless, because it's obvious, even though it is the norm. But elementary's design has quite the history of challenging the norms, isn't it?

Anyway, your suggestion is appreciated, and perhaps will make it in future versions of the app, as more and more features make it into the app and it's design becomes more and more demanding.

**Comment by NathanBnm** [comment]
You're right, I like minimalism too, but please read the elementary recommendation here https://elementary.io/fr/docs/human-interface-guidelines#window-tiles concerning windows titles.

I think you're right, but in my opinion the main title is required and important, even it's obvious, it's a reference for users. Also I think the '+' icon looks weird on the middle on the header-bar. First I didn't noticed that it was a button.

Note: It's just my personal feeling. Please don't take it as a criticism.

**Comment by aggalex** [comment]
Thank you for sharing your experience using my app. Your words have made me reconsider. I'd never thought that the '+' button looked weird if it wasn't for you. I think you're right. Plus, I'm now considering if adding a search widget on the headerbar for those who require lots of wineprefixes, which would not work with the current design.
It's experiences that matter the most, not just empty opinions.

PS: Let me assure you that what you're saying is criticism. It's constructive criticism. I (as a creator) accept constructive criticism.

###TL;DR I will move the '+' to the left and will add a window title and a search bar.

**Comment by NathanBnm** [comment]
That's true this will provide a better user experience ;)

PS: You're right I was thinking about negative criticism. Here I effectivly try to be constructive!

**Comment by aggalex** [comment]
I have tried to add the window title to the window along with the search entry, and I got to say it doesn't really work, mainly because I'm trying to make my window small. It just feels out of place.
![image](https://user-images.githubusercontent.com/11951328/53576685-ed04a980-3b7c-11e9-8924-dfa72ca2db76.png)
I don't think I'll include it. The headerbar's crimson colour is (I believe) enough to make the app recongnizable. After all, there are many elementary apps that don't have window names, like music, lookbook, etc.

**Comment by NathanBnm** [comment]
I agree with you in this case :+1:
